pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -373,13 +374,7 @@ func (ds *SQLPlugin) GetNodeSelectors(ctx context.Context,
 	callCounter := ds_telemetry.StartGetNodeSelectorsCall(ds.prepareMetricsForCall(ctx))
 	defer callCounter.Done(&err)
 
-	if err = ds.withReadTx(ctx, func(tx *gorm.DB) (err error) {
-		resp, err = getNodeSelectors(tx, req)
-		return err
-	}); err != nil {
-		return nil, err
-	}
-	return resp, nil
+	return getNodeSelectors(ctx, ds.db.databaseType, ds.db.raw, req)
```

Comment:
Queries no longer happen in a transaction, we all agree this is safe? I’m unsure w.r.t. Pagination, does it depend on the default isolation level configured?